hub/core/dataset/dataset.py

```diff
@@ -1368,16 +1385,14 @@ def append(self, sample: Dict[str, Any], skip_ok: bool = False):
             NotImplementedError: If an error occurs while writing tiles.
             Exception: Error while attempting to rollback appends.
         """
-        if isinstance(sample, Dataset):
-            sample = sample.tensors
```

Comment:
Dont remove this.

Reply:
Oops, This seems like quite the mess, Let me pull changes from hub main and redo the changes.

Reply:
Thanks for the review 😄